goserver/session/sessiontest/sessiontest.go

```diff
@@ -0,0 +1,35 @@
+package sessiontest
+
+import (
+	"errors"
+)
+
+// ErrNotFound is the error returned when something is not found.
+var ErrNotFound = errors.New("not found")
+
+// MemoryStore is an in-memory implementaion of Store.
+// It is not thread-safe and should be used by tests only.
+type MemoryStore struct {
+	items map[string]interface{}
+}
+
+// NewMemoryStore creates a new MemoryStore.
+func NewMemoryStore() *MemoryStore {
```

Comment:
The memory store is not safe for production - since it's not thread-safe. Making it a non-test package will mislead people that it can be used for production development.

Reply:
hmm, shouldn't sessiontest/ folder clear enough in that case? If not, perhaps use teststore or testsessionstore given that it implements a store?

Reply:
I didn't mean package name, I meant file name. I only suggested to rename the file from sessiontest.go to memorystore.go or something. Sorry for confusion.
Either way, if you plan to add more stuff to this file, using sessiontest.go make sense too. I assumed that it will only implement a MemoryStore.